app/uk/gov/hmrc/transitmovementseisstub/controllers/MessagesController.scala

```diff
@@ -53,6 +57,7 @@ class MessagesController @Inject() (appConfig: AppConfig, cc: ControllerComponen
   private val HTTP_DATE_FORMATTER  = DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss", Locale.ENGLISH).withZone(ZoneOffset.UTC)
   private val UUID_PATTERN         = "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$".r
   private val BEARER_TOKEN_PATTERN = "^Bearer (\\S+)$".r
+  private lazy val lrnRegexPattern = """^LRN\d*$""".r
```

Comment:
Can we make the pattern a little more obvious that it'd be a duplicate LRN, such as `^DUPLRN.*$`? I'm sure someone will test `LRN123` mindlessly one day and get a failure.